test/regression/expected/postgres_insert_select.out

```diff
@@ -0,0 +1,103 @@
+-- case: SELECT * FROM READ_CSV
+CREATE TABLE tbl (sepal_length float, variety varchar);
+INSERT INTO tbl SELECT r['sepal.length']::float, r['variety']::varchar FROM read_csv('../../data/iris.csv') r;
+SELECT sepal_length, variety FROM tbl ORDER BY sepal_length LIMIT 5;
+ sepal_length | variety 
+--------------+---------
+          4.3 | Setosa
+          4.4 | Setosa
+          4.4 | Setosa
+          4.4 | Setosa
+          4.5 | Setosa
+(5 rows)
+
+DROP TABLE tbl;
+-- case: SELECT * FROM READ_JSON
+CREATE TABLE tbl (a int PRIMARY KEY, b varchar, c real);
+INSERT INTO tbl SELECT r['a']::int, r['b']::varchar, r['c']::real FROM read_json('../../data/table.json') r;
+SELECT * FROM tbl ORDER BY a LIMIT 5;
+ a |   b    |  c  
+---+--------+-----
+ 1 | json_1 | 1.5
+ 2 | json_2 | 2.5
+ 3 | json_3 | 3.5
+ 4 | json_4 | 4.5
+ 5 | json_5 | 5.5
+(5 rows)
+
+-- DO IT AGAIN TO VALIDATE PK CONSTRAINT
+INSERT INTO tbl SELECT r['a']::int, r['b']::varchar, r['c']::real FROM read_json('../../data/table.json') r;
+ERROR:  duplicate key value violates unique constraint "tbl_pkey"
+DETAIL:  Key (a)=(1) already exists.
+DROP TABLE tbl;
+-- case: INSERT INTO PARTITION TABLE
+CREATE TABLE tbl (a int PRIMARY KEY, b text) PARTITION BY RANGE (a);
+CREATE TABLE tbl_p1 PARTITION OF tbl FOR VALUES FROM (1) TO (10);
+CREATE TABLE tbl_p2 PARTITION OF tbl FOR VALUES FROM (10) TO (20);
+INSERT INTO tbl select r['a']::int, r['b'] from duckdb.query($$ SELECT 1 a, 'abc' b $$) r;
+INSERT INTO tbl select r['a']::int, r['b'] from duckdb.query($$ SELECT 11 a, 'def' b $$) r;
+INSERT INTO tbl select r['a']::int, r['b'] from duckdb.query($$ SELECT 21 a, 'ghi' b $$) r;
+ERROR:  no partition of relation "tbl" found for row
+DETAIL:  Partition key of the failing row contains (a) = (21).
+SELECT * FROM tbl ORDER BY a LIMIT 5;
```

Comment:
Would be good to also check the contents of each partition, to double check that values go to the correct partition.

Reply:
Fixed